include/regs.h

```diff
@@ -199,4 +199,150 @@
 #define R_EN_GOROIWA_SPEED                       mREG(12)
 #define R_NAVI_MSG_REGION_ALPHA                  nREG(87)
 
+
+#define R_HREG_MODE  HREG(80) // see `HRegMode` for mode options
+
+typedef enum {
+    /*  6 */ HREG_MODE_PRINT_HILITE_INFO = 6, // print hilite information
+    /*  7 */ HREG_MODE_UCODE_DISAS, // various controls for the ucode disas system
+    /*  8 */ HREG_MODE_PRINT_MEMORY, // print memory at a specified address
+    /*  9 */ HREG_MODE_UNUSED_9,
+    /* 10 */ HREG_MODE_PLAY, // various debug controls for the Play state
+    /* 11 */ HREG_MODE_PERSPECTIVE, // set various properties of the view perspective
+    /* 12 */ HREG_MODE_INPUT_TEST, // displays inputs from the specified controller
+    /* 13 */ HREG_MODE_SCHED, // toggle various vi special features
+    /* 14 */ HREG_MODE_UNUSED_14,
+    /* 15 */ HREG_MODE_PRERENDER,
```

Comment:
```suggestion
    /*  8 */ HREG_MODE_PRINT_MEMORY, // print memory at a specified address
    /* 10 */ HREG_MODE_PLAY = 10, // various debug controls for the Play state
    /* 11 */ HREG_MODE_PERSPECTIVE, // set various properties of the view perspective
    /* 12 */ HREG_MODE_INPUT_TEST, // displays inputs from the specified controller
    /* 13 */ HREG_MODE_SCHED, // toggle various vi special features
    /* 15 */ HREG_MODE_PRERENDER = 15,
```
Maybe worth skipping the unused ones?

Reply:
I find skipping to only be worth when its large gaps. otherwise its messier to maintain in a modding scenario (imo)

Reply:
Sounds good